### ISSUE TITLE
Multiple icons in icon tray

### ISSUE THREAD

**Issue by AwryWolf** [xenolightning/AudioSwitcher_v1#2181]
I was wondering why I get multiple icons appearing in the icon tray? I tried to uninstall the programme and reinstall but found no way to uninstall?

**Comment by xenolightning** [comment]
I see you've removed the issue template. Can you fill out the issue template, and maybe attached a screenshot of the problem?

**Comment by AwryWolf** [comment]
Version 1.8.0.142 
Windows 8.1 64-bit
Logitech Speakers line in and USB platonic headset

Sorry about not setting this, I quickly rushed out a thread as I was busy at the time. I greatly apologize as I know how little use it was to you having no information.

Again, I am more curious how to uninstall the programme properly so I can reinstall. The programme doesn't appear in my add or remove programme list. I imagine this could solve the rogue sprites icons. See attached
![untitled](https://user-images.githubusercontent.com/24205770/43004768-af55a8ae-8c28-11e8-86e2-79c371363b66.png)

**Comment by AwryWolf** [comment]
Fixed it by getting rid of old tray icons and restarting explorer.exe. Then reinstalled.

**Comment by xenolightning** [comment]
Weird, maybe it was Windows doing something strange.

To "uninstall" you just delete the exe, there is no installer.

**Comment by AwryWolf** [comment]
Yea, I basically just went into regedit and located the tray and just deleted the two old tray files. Then just closed explorer.exe and restarted. See I deleted the exe but when I did a system search there were several rogue files remaining for audio switcher. Text files etc. I deleted those, I imagine they were just leftover files. Reinstalled it not and its fine apart from sometimes it just doesn't start on windows startup despite settings being set. Idk.
